qa/os/src/test/java/org/elasticsearch/packaging/test/EnrollmentProcessTests.java

```diff
@@ -27,7 +26,6 @@
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assume.assumeTrue;
 
-@AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/79810")
 public class EnrollmentProcessTests extends PackagingTestCase {
```

Comment:
I would suggest a test (or two) where the second node tries a "wrong" enrollment token, and then tries an enrollment token with some obviously wrong addresses first in th list, before finally succeding.

Only add if you believe it ads value (When testing manually, initially I wasn't sure if I could try again with another enrollment token on the same instalation).